lib/command/src/Obelisk/Command.hs

```diff
@@ -236,6 +236,9 @@ thunkConfig = ThunkConfig
 thunkUpdateConfig :: Parser ThunkUpdateConfig
 thunkUpdateConfig = ThunkUpdateConfig
   <$> optional (strOption (long "branch" <> metavar "BRANCH" <> help "Use the given branch when looking for the latest revision"))
+#if MIN_VERSION_nix_thunk(0,7,1)
+  <*> optional (strOption (short 'r' <> long "rev" <> metavar "REVISION" <> help "Update to this specific revision"))
+#endif
```

Comment:
I backported for now, but shouldn't we simply be importing all these parsers from nix-thunk? 
@ryantrinkle